wagtail-{{ cookiecutter.project_name_kebab }}/pyproject.toml

```
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "wagtail-{{ cookiecutter.project_name_kebab }}"
+authors = [{name = "{{ cookiecutter.full_name }}", email = "{{ cookiecutter.email }}"}]
+description = "{{ cookiecutter.project_short_description }}"
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    {% if cookiecutter.open_source_license == 'MIT license' -%}
+    "License :: OSI Approved :: MIT License",
+    {%- elif cookiecutter.open_source_license == 'BSD license' -%}
+    "License :: OSI Approved :: BSD License",
+    {%- elif cookiecutter.open_source_license == 'ISC license' -%}
+    "License :: OSI Approved :: ISC License (ISCL)",
+    {%- elif cookiecutter.open_source_license == 'Apache Software License 2.0' -%}
+    License :: OSI Approved :: Apache Software License",
+    {%- elif cookiecutter.open_source_license == 'GNU General Public License v3' -%}
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    {%- elif cookiecutter.open_source_license == 'Not open source' -%}
+    "License :: Other/Proprietary License",
+    {% endif %}
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Framework :: Django",
+    "Framework :: Django :: 3.0",
+    "Framework :: Django :: 3.1",
+    "Framework :: Django :: 3.2",
+    "Framework :: Wagtail",
+    "Framework :: Wagtail :: 2",
```

```suggestion
    "Framework :: Wagtail :: 4",
```

at the very least (#28 will add it)

---

Handled in the rebased 169a36c